content/2016-12-26_komunitni-python-stanek-na-linuxdays.md

```diff
@@ -0,0 +1,90 @@
+Což byl opravdu skvělý tah. Když už se nám ráno před samotnou akcí povedlo shromáždit vše a všechny, zjistili jsme, že jsme jeden z největších stánků co do rozlohy i do počtu prezentujících.
+Na stánku jsme totiž měli několik her na zelených destičkách typu NodeMCU, micro:bit a na nich hry, které poháněl MicroPython.
+O tyto destičky byl velký zájem a tak tímto velmi děkuji Petru Viktorinovi, Lumíru Balharovi a Janu Čermákovi, že mě v tom nenechali a všem nadšencům ochotně vysvětlovali a flashovali jejich destičky.
+
+Zmínění Roudničáci si přivezli her asi 10 a kdyby jim večer nevypli proud, sedí tam ještě teď. Na Adamovi, Jakubovi a Jardovi z Roudnice je super, že dokážou okolí nejen pobavit, ale i proškolit a aby někomu nepřišlo druhý den, že mají stejný program, tak 
+kluci přes noc ještě pár her doprogramují. Skvělým příkladem toho byla hra rádoby „GTAčko“, ke které přes noc Adam kromě dosavadního volantu s akcelerátorem ještě pedál fungující na světla z auta nalezená na půdě.
+Velké nadšení vzbudila jejich hra s vlaječkami na rychlost, digitální tabule na kterou se dá kreslit či hrát had dálkovým ovládáním. Rádoby „GTAčko“ s pedálem, volantem a světly. Jednoznačně parádní show a u našeho stánku byl téměř pořád plno.
+
+![Rádoby "GTAčko"]({filename}/images/linuxdays_img_2194.jpg){:width="350px" :center}
+
+![Hry na stánku měly velký úspěch]({filename}/images/linuxdays_img_2211.jpg){:width="350px" :center}
+
+![Jarda Vysoký s digitální tabulí, kterou naprogramoval s klukama z Gymnázia Roudnice nad Labem]({filename}/images/linuxdays_img_2216.jpg){:width="350px" :center}
+
+
+Při té příležitosti jsme lidem povídali o tom, co vše v české Python komunitě děláme. Překvapivé pro nás bylo, že se na konferenci objevilo spousty lidí, kteří nikdy ani o python.cz neslyšeli, na Pyvo srazech nebyli a celkově byli nadšení, že se o tom mohli dozvědět, protože v Pythonu dělají.
+Dále jsme samozřejmě prezentovali kromě python.cz i PyLadies, Advanced PyLadies, Blog Python komunity, PyCon v Brně.
+Na stánku nechybělo ani Python žezlo, 3D vytištěné od Miro Hrončoka a Jakuba Průši. Děkujeme za jeho propůjčení. 
+
+![Miro Hrončok a jeho Python žezlo samozvané Pyč #pythonicstaffofenlightenment]({filename}/images/linuxdays_img_2158.jpg){:width="350px" :center}
+
+Chvílemi se k nám přidávali i prezentující z ostatních spřátelených stánků, jako je například CZ.NIC. Python nadšenců bylo na této konferenci opravdu dost. Došlo i k výměně věciček na bastlení po kterých jsme už nikterak netoužili za knihu o GIT do Python knihovničky, po které jsme naopak hrozně toužili.
+
+![Přátelství s ostatními stánky na nás bylo znát]({filename}/images/linuxdays_img_2200.jpg){:width="350px" :center}
+
+Podobný stánek jsme o pár týdnů déle postavili i na PyCon 2016 v Brně, kam se také přijeli podívat kluci z Roudnice i se svou tabuli plnou her. 
```

Comment:
Promiň za tu reakci, ale od České Třebové na východ se "dýl" neodpouští. Máme to v krvi a neumíme se přes to přenést 😢